packages/openneuro-app/src/scripts/errors/errorBoundary.jsx

```diff
@@ -142,16 +143,17 @@ class ErrorBoundary extends React.Component {
   }
 
   render() {
+    const error = this.props.error || this.state.error
```

Comment:
we should reverse the order of assignment here to `this.state.error || this.props.error`